.github/workflows/runSchematronValidation.yml

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Run Schematron Validation
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on: [workflow_dispatch]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This job runs schematron validation(s) on the file that is passed in 
+  run-schematron-validation:
+   # Run on ubuntu
+    runs-on: ubuntu-latest
+
+    # Checkout repository and its submodules
+    steps:
+      # Check-out the repository under $GITHUB_WORKSPACE
+      - uses: actions/checkout@v2
+        
+      # Check-out submodules
+      - name: Checkout submodules
+        uses: textbook/git-checkout-submodule-action@2.1.1
+        with:
+          remote: true
+
+      # Install Saxon HE to /tmp
+      - name: Install Saxon HE
+        run: |
+          echo "Installing Saxon"
+          mkdir -p /tmp/saxon
```

Comment:
So we are moving away from the `mvn` approach and caching things in `~/.m2` within the context of the container user?

[For reference](https://github.com/ohsh6o/skyrim-schematron-tutorial/blob/main/.github/workflows/report.yml#L37-L41)